pos_product_search_gs1/readme/DESCRIPTION.md

```diff
@@ -0,0 +1 @@
+This module enhances the POS (Point of Sale) system by implementing barcode scanning functionality for product search.
```

Comment:
Please add USAGE
In the point of sale product list screen, clic in the search bar and scan the GS1 code